src/trio/_core/_tests/test_mock_clock.py

```diff
@@ -92,7 +92,7 @@ async def test_mock_clock_autojump(mock_clock: MockClock) -> None:
     mock_clock.autojump_threshold = 0
     # if the above line didn't take affect immediately, then this would be
     # bad:
-    await sleep(100000)
+    await sleep_forever()
```

Comment:
This isn't correct, the code is testing that the autojump clock is able to skip a large (but finite) amount of sleeping.

Reply:
Fixed in new version